### DISCUSSION TITLE
error 'module not found' when trying to create remix project

### DISCUSSION THREAD

**Discussion by ArdieWijaya** [remix-run/remix#6385] [Q&A]
hello everyone, I tried to create a remix project using this command
`npx create-remix@latest`
but I got errors everytime I try to create it. I attach the error message below

`npx create-remix@latest
internal/modules/cjs/loader.js:892
  throw err;
  ^

Error: Cannot find module 'node:child_process'
Require stack:
- /Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/config.js
- /Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/compiler/utils/postcss.js
- /Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/compiler/plugins/cssImports.js
- /Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/compiler/css/compiler.js
- /Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/compiler/compiler.js
- /Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/compiler/build.js
- /Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/cli/commands.js
- /Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/cli/run.js
- /Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/cli/index.js
- /Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/index.js
- /Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/create-remix/dist/cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:889:15)
    at Function.Module._load (internal/modules/cjs/loader.js:745:27)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (/Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/config.js:15:26)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Module.require (internal/modules/cjs/loader.js:961:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/config.js',
    '/Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/compiler/utils/postcss.js',
    '/Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/compiler/plugins/cssImports.js',
    '/Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/compiler/css/compiler.js',
    '/Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/compiler/compiler.js',
    '/Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/compiler/build.js',
    '/Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/cli/commands.js',
    '/Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/cli/run.js',
    '/Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/cli/index.js',
    '/Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/@remix-run/dev/dist/index.js',
    '/Users/ardiewijaya/.npm/_npx/5164864a48bff686/node_modules/create-remix/dist/cli.js'
  ]
}`

I already tried to reinstall create-remix but it didn't work too

**Comment by zfrankdesign** [comment]
I got/am getting this same error.. Using `asdf` to manage node versions, `node 15.14.0` and `npm 7.7.6`

UPDATE:
It turned out to be my node and npm versions listed above. I moved over to a new folder with a 18.x local version of node installed and things seem to work.
